docs/tutorial.md

```diff
@@ -160,6 +161,107 @@ By specifying hostname, the guestbook service will only be available on the spec
 
 Now the `guestbook` application will be available on both HTTP and HTTPS only on the specified host (`<guestbook.contoso.com>` in this example).
 
+### Certificate issuance with Lets Encrypt
+
+This section configures [Lets Encrypt](https://letsencrypt.org/) to issue the certificate that Applicaiton Gateway will use for SSL/TLS termination at the gateway.  This uses the Kubernetes add-on [cert-manager](https://github.com/jetstack/cert-manager) to automate the creation and management of the kubernetes secret containing the certificate.
+
+> [!NOTE]
+> The following steps will install `cert-manager` in your cluster, full documnetation can be found on the cert-manager docs site [here](https://docs.cert-manager.io)
+
+1. run this script to install the `cert-manager` helm chart [(from docs.cert-manager.io)](https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html#steps)
+
+
+```sh
+# Install the CustomResourceDefinition resources separately
+kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.8/deploy/manifests/00-crds.yaml
+
+# Create the namespace for cert-manager
+kubectl create namespace cert-manager
+
+# Label the cert-manager namespace to disable resource validation
+kubectl label namespace cert-manager certmanager.k8s.io/disable-validation=true
+
+# Add the Jetstack Helm repository
+helm repo add jetstack https://charts.jetstack.io
+
+# Update your local Helm chart repository cache
+helm repo update
+
+# Install the cert-manager Helm chart
+helm install \
+  --name cert-manager \
+  --namespace cert-manager \
+  --version v0.8.0 \
+  jetstack/cert-manager
+```
+
+2. Create a `ClusterIssuer`, these details are required by `cert-manager` to automatically generate a certificate request which is fulfilled by the issuer (`Lets Encrypt`).  `Let’s Encrypt` uses the ACME protocol to verify that you control a given domain name and to issue you a certificate
```

Comment:
Where is the `ClusterIssuer` resource suppose to reside ?

Reply:
I'll add some text about the namespaces Issuers can reside in, and add a reference to the Issuer cert-manager help page that documents all the supported properties